packages/protocol/lib/compatibility/verify-bytecode.ts

```diff
@@ -34,7 +34,8 @@ interface VerificationContext {
 // Checks if the given transaction is a repointing of the Proxy for the given
 // contract.
 const isProxyRepointTransaction = (tx: ProposalTx, contract: string) =>
-  tx.contract === `${contract}Proxy` && (tx.function === '_setImplementation' || tx.function === '_setAndInitializeImplementation')
+  tx.contract === `${contract}Proxy` &&
```

Comment:
I don't think this will be relevant anymore, as this function is used for finding the implementation contract, which we would now do via the registry setting